client/google-search-app/src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,13 @@
+.footer {
+  position: fixed;
+  bottom: 0;
```

Comment:
avoid position fixed in next PR.
example: https://www.w3schools.com/css/tryit.asp?filename=trycss_position_fixed